### PR TITLE
CGS-411: Security fixes for npm dependencies

### PR DESCRIPTION
**Pull request by vrjuliao**
## Description of the change
All the vulnerabilities classified as `high` or `critical` were solved.
The following packages were upgraded to the following versions:
- `node-fetch`: 2.6.4
- `cross-fetch`: 3.1.15
- `async`: 2.6.7
- `trim`:  0.0.3

These changes were done by changing  the `docs/yarn.lock` file directly, however, for `trim` upgrade it demands the inclusion of the field "resolutions"  at `docs/package.json`.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Security fix

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] Jira issue referenced in commit message and/or PR title

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -51,5 +51,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "resolutions": {
+    "trim": "^0.0.3"
```
Comment (VictorGFM) [inline docs/package.json:56]:
```suggestion
    "trim": "^0.0.3",
    "async": "^2.6.4",
    "cross-fetch": "^3.1.5"
```

**Review by VictorGFM — COMMENTED** [review]
Hey Vinicius, thanks for fixing these vulnerabilities. I noticed that there are a lot of changes in the `yarn.lock` file that are not necessarily related to the vulnerable dependencies that we want to update in this case, also, some of these changes update some components and the way they are displayed in the UI (See images below), normally we want to preserve the design and behavior of the application. Thus, I believe the simplest way to strictly update the vulnerable dependencies here is by just adding them to the `resolutions` block in the `package.json` and then updating them by running a `yarn install`. Could you please do that and make sure that only these dependencies get updated in the yarn.lock file and then test if nothing breaks after that? (Notice that by updating the `cross-fetch` dependency the `node-fetch` will also be updated, so it's optional to add it to `resolutions` in this case)

**Current Approzium Doc**
image

**Updated Approzium Doc**
image

**Comment by vrjuliao** [comment]
Hi @VictorGFM. Thank you for reviewing the changes. I noticed that I forgot to roll back some `yarn.lock` changes that were done when I was trying to update `docusaurus`. So, now all the changes are consistent.
About the "resolutions" field at `package.json`, actually, we only have to keep the "trim" there. The another dependencies can be persistently updated directly in `yarn.lock` file.

**Review by VictorGFM — APPROVED** [review]
LGTM

**Comment by VictorGFM** [comment]
@wcmjunior Could please merge this PR? I actually don't have permission for that.

**Pull request merged**